pkgs/development/libraries/LAStools/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, cmake, fetchurl}:
+stdenv.mkDerivation rec {
+
+  version = "180812";
+  name = "LAStools-${version}";
+
+  src = fetchFromGitHub {
+    owner = "LAStools";
+    repo = "LAStools";
+    rev = "162cf032f25cac492712a8568a08b224a5fb40c2";
+    sha256 = "117c9csbfwsx424ha695l7d99cswi316k8q338mzk1lxlk8p3xbz";
+  };
+
+  nativeBuildInputs = [cmake];
+
+  meta = {
+    description = "Efficient tools for LiDAR processing.";
+    homepage = https://www.laszip.org;
```

Comment:
Isn't it http://lastools.org?

From their [README](https://github.com/LAStools/LAStools/blob/master/README.txt):

> latest updates:  http://lastools.org